### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by drallcom3** [zulfajuniadi/unity-ecs-navmesh-mapbox#3]
Hello,

using the latest 2018.3 I get various errors like

Assets\NavJob\Systems\NavAgentAvoidanceSystem.cs(138,33): error CS0115: 'NavAgentAvoidanceSystem.OnCreateManager(int)': no suitable method found to override

or

Assets\NavJob\Systems\NavAgentSyncSystem.cs(21,99): error CS0246: The type or namespace name 'TransformMatrix' could not be found

**Comment by MostHated** [comment]
Quite a few things changed in 2018.3, so I would expect a lot of things are not going to work properly, especially so early in the 2018.3 beta. There was even a breaking change going from 2018.2.7f1 to just 8f1. They change a lot of things very often when it comes to ECS and jobs. It is hard to keep up with examples and tutorials. 

For this to work, I had to just revert back down to 7f1 as OnCreateManager() no longer accepts a parameter at all and TransformMatrix no longer exists in newer code, it has been changed to LocalToWorld.
